### ISSUE TITLE
Feature Requests: Custom Button Card Like State and Mushroom Chip Card Like Values

### ISSUE THREAD

**Issue by efaden** [Clooos/Bubble-Card#239]
I love the bubble card, but there are two things that would be neat to add.

Firstly custom button card allows you to define the icon, color, etc based upon the state/value of the an entity (not even the entity the button controls).  So for example I could have a card for my fireplace that has an icon without flames and is grey when on and an icon with a flame that is red when it is on.  It would just be cool to be able to really customize the buttons that way.

My second request would be some sort of a card (could be a button with no action) that allows you to display a state/value.  An example would be the temperature, or an icon that can change color to note motion/occupancy.  So for example I have a popup for controlling the basement lights.  It would be slick to be able to throw an icon in the header or somewhere that shows an icon of a person that changes color depending on state of a motion sensor.

**Comment by awilliams84** [comment]
I agree on the state or template ability in the popup buttons. Is this possible??

I use mushroom cards for buttons currently at the top that show a color, icon, and content based on states. For example: it counts the number and types of doors and windows open in my house and if I have one door open, the icon will be an open door, color yellow, and content "1".
